frontend/src/components/Overview.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Overview(){
```

Comment:
Delete unused component.
If you need it for future features add the component in the related PR